src/common/components/Input/index.tsx

```diff
@@ -145,15 +145,13 @@ export class Input extends React.Component<InputProps, InputState> {
                     <InputLabel htmlFor={this.state.id} error={showError}>
                         {label}
 
-                        {description && (
-                            <Description description={description} />
-                        )}
+                        {description && <Description description={description} />}
                     </InputLabel>
                 )}
 
                 <InputElement id={this.state.id} onBlur={this.makeDirty} hasError={showError} aria-invalid={showError} {...rest} />
 
-                <Collapse isOpen={showError && typeof error === 'string'} className={`${inputErrorTransition}`}>
```

Comment:
@jonathanp Is there a reason we did a typeOf check here? I can't see why this shouldn't work just fine supporting an element